### DISCUSSION TITLE
GitHub Support does not respond after 3 weeks; TICKETS JUST GET CLOSED WITHOUT A RESPONSE!!

### DISCUSSION THREAD

**Discussion by ghost** [community/community#134334] [New to GitHub]
**I've been contacting GitHub Support for the past 3 weeks; unlike most of everyone else, GitHub Support sees my tickets, BUT JUST CLOSES THEM WITHOUT RESPONDING??**

My account (Saharsha-N, now no longer visible on GitHub) was suspended without reason: no email, no GitHub Support message, nothing. And with every ticket I create to ask GitHub Support to detail how I violated the TOS, it just gets closed within 2-3 days.

The community seems to have similar, but different problems. No online forum suggested by GitHub Support or in the general community seem to talk about this issue.

How long did it take for GitHub Support to actually respond on GitHub? Does GitHub Support closing my tickets without a response signify anything?

**Comment by deep-sengupta** [comment]
How long did it take for GitHub Support to actually respond on GitHub?
GitHub Support usually responds within a few days to a week.

Does GitHub Support closing my tickets without a response signify anything?
Yes, it might mean:

Automated Flagging: Your tickets are being flagged and closed automatically.
Policy Violation: There may have been a serious violation of GitHub's rules.
Issue Considered Resolved: GitHub might think the issue is resolved or not open for discussion.
In any case, the lack of communication is unusual and suggests a problem with their support process in your situation.

**Reply by ghost** [reply]
I don't know the extent to which I violated GitHub's rules, but I can confidently say I haven't done anything suspicious on my account; I've only developed some side projects, stored code from research, and website code from school projects and my startups.

I even tried calling GitHub support with their phone number, but for account-related inquiries, the automated phone bot says to open a GitHub support ticket and follow up from there. Is there any other way I can get in contact with GitHub Support to escalate my issue or even get an explanation of what term(s) I violated in the first place?

**Reply by deep-sengupta** [reply]
Tweet @GitHubHelp: Mention your issue and ticket number.
Reach out on LinkedIn: Contact GitHub employees in support.

@GitHub @GitHubSupport My account (*****) was suspended without any notice or explanation. My support tickets are being closed without responses. I haven't violated any rules to my knowledge. Can someone please assist me in resolving this? #GitHubSupport

**Reply by ghost** [reply]
Thank you for the tip! I will be sure to do that, hopefully I will get a response this way.

**Comment by ozill87** [comment]
Good morning

**Comment by ghost** [comment]
Is anyone else having a problem like this?

**Comment by digitaldrreamer** [comment]
Been 79 days and I haven't gotten any response from Github Support in my ticket. #3734134

Real shitty. I'm thankful I didn't wait up on them and started working on something else.

**Comment by KibitoHagoromo** [comment]
Could you please escalate my issue Ticket 4388902 that is still unsolved for one month since the ticket creation.

Best Regards.
